src/main/java/snowy/Snowy.java

```diff
@@ -0,0 +1,62 @@
+package snowy;
+
+import snowy.common.Command;
+import snowy.common.CommandResult;
+import snowy.data.SnowyException;
+import snowy.storage.Storage;
+import snowy.tasklist.TaskList;
+import snowy.ui.TextUi;
+import snowy.parser.Parser;
```

Comment:
I like how all your imports are specific and are arranged in alphabetical order.